addon/serializers/osf-serializer.js

```diff
@@ -14,14 +16,63 @@ export default DS.JSONAPISerializer.extend({
         }
     },
 
+    _restoreLinksBeforeEmbed(resourceId, embeddedType) {
+        // If items are embedded, the embedded resource detail replaces the links. The APIv2 might need to be modified
```

Comment:
@samchrisinger We talked yesterday - someone is going to need to add it.  Currently, if you are embedding a resource, the link used to fetch it is replaced. Sometimes we need this link.  For example, if we have a page displaying collections and we've embedded linked_nodes so we can show the projects in the collection, if we go to add a project to the collection we don't have the links we need. Ticket here https://openscience.atlassian.net/browse/OSF-6488.